staging/src/k8s.io/apimachinery/pkg/runtime/swagger_doc_generator.go

```diff
@@ -194,6 +197,33 @@ func ParseDocumentationFrom(src string) []KubeTypes {
 	return docForTypes
 }
 
+// GetFieldAndDocFrom gets all types' field and documentation and returns them as an
+// array. Each type is again represented as an map from field to its name and doc.
+// Embedded fields and fields with no name in tag will be excluded.
```

Comment:
Does this mean no name in a json tag? So that would be fields that are considered internal right?

Reply:
Yes, they may be internal so we don't need to check for these docs, or like TypeMeta which is inline, should we also check it (it isn't in any API groups)?

Reply:
No I think that's correct, I just wanted to clarify my understanding. Might be helpful to include this detail in the godoc here to help others who have the same question in the future, but otherwise, no problem